lib/matplotlib/axes/_axes.py

```diff
@@ -3958,6 +3958,7 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
 
         See Also
         --------
+        .Axes.bxp : Draw a boxplot based on statistical parameters (instead of data).
```

Comment:
```suggestion
        .Axes.bxp : Draw a boxplot based on pre-computed statistics (instead of raw data).
```
?

Reply:
Thanks @rcomer. This is a good wording. 

@story645, I've removed the instead-comment for this direction. While I see the appeal of "Computes boxplot statistics and draws a boxplot", I'd like to keep the "Draw ..." pattern because that's the primary objective. And I believe people think "here's my data, make a boxplot". It's implicitly understood that you have to calculate statistics, but explicitly mention that may even be distracting.

Reply:
It's because the computed stats [can be data](https://xkcd.com/2582/) that I thought the "raw" would help but yes I appreciate that point.  Pretty much none of the data I analyse would be considered raw by an instrument scientist.

Reply:
@story645 please check whether the current wording works for you:

#### Summary lines:
- `boxplot()`: Draw a box and whisker plot.
- `bxp()`: Draw a box and whisker plot based on pre-computed statistics.

*Motivation:* Since it's very rare that people have pre-computed statistics, `boxplot()` (including computation) is the canonical behavior and we don't need to qualify the type of input. `bxp()` has the addition "base on pre-computed statistics" because that special and needs explicit mentioning.

#### See also:
- `boxplot()` -> `bxp()`: Draw a boxplot based on pre-computed statistics.
- `bxp()` -> `boxplot()`: Draw a boxplot based on raw data (instead of pre-computed statistics).

*Motivation:* The first is just the summary line of `bxp()` (mentioning what's special). The second has the extended explanation, because the docstring is in the context of the `bxp()` special case, so just an unqualified "Draw a boxplot" seems not sufficient - we have to explicitly state what's different. I know, you nitted "raw", but I haven't found a better term, and I think with the addition "raw data (instead of pre-computed statistics)" it's clear enough.

Reply:
Done.